runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/internal/event/EngineEventFormatter.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2021-2023 Aklivity Inc.
+ *
+ * Aklivity licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.aklivity.zilla.runtime.engine.internal.event;
+
+import org.agrona.DirectBuffer;
+
+import io.aklivity.zilla.runtime.engine.Configuration;
+import io.aklivity.zilla.runtime.engine.event.EventFormatterSpi;
+import io.aklivity.zilla.runtime.engine.internal.types.event.EngineConfigWatcherFailedExFW;
+import io.aklivity.zilla.runtime.engine.internal.types.event.EngineEventExFW;
+import io.aklivity.zilla.runtime.engine.internal.types.event.EventFW;
+
+public final class EngineEventFormatter implements EventFormatterSpi
+{
+    private static final String CONFIG_WATCHER_FAILED_FORMAT =
+        "Please restart to apply any config changes.";
```

Comment:
```suggestion
        "The initial config loaded successfully, but the config watcher could not be created. Restart Zilla to apply any new config changes.";
```
It feels a bit wordy but will hopefully keep someone from getting into a restart loop.

Reply:
Turns out that the config is not yet loaded successfully when this happens, leading to the above message followed by config validation failures for malformed `zilla.yaml` syntax.

Propose we go with this instead.
```suggestion
        "Dynamic config reloading is disabled.";
```
Then when an `IOException` occurs, we also have the exception message, such as:
```
ENGINE_CONFIG_WATCHER_FAILED Dynamic config reloading is disabled. User limit of inotify instances reached or too many open files.
```